sp_dev.py

```diff
@@ -109,10 +109,23 @@ def get_bytearray_str(bytes_array):
         excerpt += "]"
         res = res + " " + excerpt
     return res
+
+def get_dataset_str(dataset):
+    columns = [f"{col}({DATATYPES_STR[col_type]})"
+               for (col, col_type)
+               in zip(dataset.columns, dataset.types)]
+    rows = [tuple([get_typed_value_str(col_type, row.elements[idx])
+                   for (idx, col_type) in enumerate(dataset.types)])
+            for row in dataset.rows]
+    res = f"Dataset {len(dataset.rows)}x{len(dataset.types)}:\n"
+    res += f"\t\tColumns: {columns}\n\t\tRows: {rows}"
```

Comment:
I would suggest to use a new statement for each new line(s) that are displayed to the end user.
```suggestion
    res = f"Dataset {len(dataset.rows)}x{len(dataset.types)}:\n"
    res += f"\t\tColumns: {columns}\n"
    res += f"\t\tRows: {rows}"
```

Reply:
Done